molo/commenting/templates/notifications/tags/notification_banner.html

```diff
@@ -1,10 +1,16 @@
 {% load i18n %}
-<div class="notifications-list__item">
```

Comment:
@Mitso the tests are currently failing because we searched for notifications against this class.